rts/lra/run.sh

```diff
@@ -4,7 +4,6 @@ function finish {
 
     rm -rf $NARAYANA_INSTALL_LOCATION
 }
-trap finish EXIT
```

Comment:
Here the finish function was called twice, 1st time when `finish` was called it killed all the running processes. And 2nd time `finish` was called when trapping EXIT, there were no process to kill, so the build was failing.

Reply:
What happens if the script finishes before `finish` is called? Will the script still clean up after itself?

Reply:
Before script exits, everything is getting cleaned up only when the finish is called for first time [here](https://github.com/jbosstm/quickstart/blob/master/rts/lra/run.sh#L95).

Reply:
No I mean that if you remove the trap call and the bash script fails before reaching the call to `finish` how do the java processes which were started get cleaned up?

Reply:
Okay, @mmusgrov I got your point, so instead of removing `trap` command, I will add few `if` conditions in the `finish` function

Reply:
No. Just make sure the finish command doesn't try to kill the same process twice - ie make it idempotent.

Reply:
I am removing the `finish` call from the last line. The `finish` that is called by `trap` will do the cleanup. Currently the build will fail because before exiting the script, last command i.e. `curl` is failing due to the recovery issue.

And earlier the build was passing because the last command `finish` that was successfully executed before exiting the script.